### ISSUE TITLE
Different result when change isStatic after creating body.

### ISSUE THREAD

**Issue by greenSnot** [liabru/matter-js#319]
https://jsfiddle.net/gf3jLvjt/1/

compare to remove line 29, the result is quite different. I'm not sure if it is caused by svg's issues.
` comp.bodies[0].isStatic = true;`

**Comment by greenSnot** [comment]
My goal is to create some static bodies and then these bodies can be non-static by some conditions.
To do this way
```
      let body = Bodies.fromVertices(100 * i + 300, 200 + i * 30, vertexSets, {
        isStatic: false,
      }, true, 0.01, 30);
      body.isStatic = true;
```
because if I just set {isStatic: true} then after Engine.run() and change this property this body will be out of screen(by some force?) immediately.

**Comment by greenSnot** [comment]
https://jsfiddle.net/gf3jLvjt/4/

And look at line 20, it's ok for 0.3
`      vertexSets.push(Vertices.scale(points, 0.3, 0.3));`
But It's bad result for 0.4 and the svg path has some flaws
`      vertexSets.push(Vertices.scale(points, 0.4, 0.4));`

**Comment by liabru** [comment]
You need to use `Body.setStatic(body, true)`. Though note that you can't currently set a body back to dynamic after it has been set to static (but this should be solved soon see #187).
